spec/system/viewing_reviewable_spec.rb

```diff
@@ -143,11 +145,80 @@
         expect(page).to have_current_path("/")
       end
 
-      it "allows to lookup IP" do
-        SiteSetting.reviewable_old_moderator_actions = true
+      it "shows IP lookup information when insights tab is viewed" do
+        user = reviewable_flagged_post.target_created_by
+        user.update!(ip_address: "81.2.69.142")
```

Comment:
I think we can set the user at the top-level fabricators. Something like:

```ruby
  fab!(:user} { Fabricate(:user, ip_address: "81.2.69.142")
  fab!(:reviewable_flagged_post) { Fabricate(:reviewable_flagged_user, created_by: user) }
```

It is ultra minor but then we will not need to repeat it later.

Also maybe stub can go to before block 
```ruby
  DiscourseIpInfo.open_db(File.join(Rails.root, "spec", "fixtures", "mmdb"))
        Resolv::DNS
          .any_instance
          .stubs(:getname)
          .with("81.2.69.142")
          .returns("ip-81-2-69-142.example.com")
```